File: src/main/scala/unyo/Env.scala
package unyo

import java.awt.{Toolkit,Dimension}

object Env {
  private val tk = Toolkit.getDefaultToolkit
  val frameWidth  = tk.getScreenSize.getWidth.toInt  * 2 / 3
  val frameHeight = tk.getScreenSize.getHeight.toInt * 2 / 3
}

File: src/main/java/unyo/Env.java
package unyo;

import java.awt.Toolkit;
import java.awt.Dimension;

public class Env {
    private Env() {}

    public static int frameWidth() {
        return (int)(Toolkit.getDefaultToolkit().getScreenSize().getWidth() * 2 / 3);
    }

    public static int frameHeight() {
        return (int)(Toolkit.getDefaultToolkit().getScreenSize().getHeight() * 2 / 3);
    }
}
